src/core/labeling/qgstextlabelfeature.h

```diff
@@ -20,6 +20,7 @@
 #include "qgslabelfeature.h"
 #include "qgstextdocument.h"
 #include "qgstextmetrics.h"
+#include "qgspallabeling.h"
```

Comment:
@nyalldawson , is that additional include required? Just checking.

Reply:
Yes -- it was previously indirectly included through one of the other includes, now the includes are tightened up and the other file doesn't include it.